client/src/app.js

```diff
@@ -217,6 +185,45 @@ export function checkStreamResult(result) {
     }
 }
 
+function handleConnectionStateChange(rtcConn) {
+    if(rtcConn.connectionState === SUCCESSFUL_CONNECTION) { 
+        updateState({ established: true }); 
+    }
+
+    if(rtcConn.connectionState == DISCONNECTED || 
+    rtcConn.connectionState == FAILED_CONNECTION) {
+        updateState({ 
+            established: false,
+            isPlaying: false
+        });
+    }
+}
+
+function setMediaDiscription(ev) {
+    // data channel to recieve the media title
+    var channel = ev.channel;
+    channel.onmessage = function(event) {
```

Comment:
I think this is the function that should be named. We could call it onDataChannelMessage and the param can be messageEvent.